src/wwwroot/app/main.scss

```diff
@@ -6,11 +6,17 @@
 @import "./styles/secondary-menu";
 @import "./styles/side-menu";
 @import "./styles/sidebar";
+@import "./styles/text-editor";
```

Comment:
oh, why we use double quote in js? Please, speak with @msvaillant. We need check it with eslint.